packages/venia-concept/package.json

```diff
@@ -98,7 +98,7 @@
     "sharp": "~0.21.2"
   },
   "engines": {
-    "node": "10.x",
+    "node": ">=10.x",
```

Comment:
Can you leave this one more restrictive? We don't want to support the actual consuming app for infinite forward versions. That's too much that can go wrong.

The library code, on the other hand, can totally use the version string you suggest.

Reply:
```suggestion
    "node": "10.x",
```